config/environment.rb

```diff
@@ -3,3 +3,13 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActionMailer::Base.smtp_settings = {
+  user_name: Rails.application.secrets.sendgrid_username,
+  password: Rails.application.secrets.sendgrid_password,
+  domain: 'getguesstimate.com',
```

Comment:
don't like this domain getting repeated everywhere.  Can it be referenced from the app secrets or something?